pkg/server/server.go

```diff
@@ -603,19 +603,7 @@ func (s *Server) generatePolicyRulesForPodAndFamily(pod *v1.Pod, podInfo *contro
 			}
 		}
 
-		var ingressEnable, egressEnable bool
-		if len(policy.Spec.PolicyTypes) == 0 {
-			ingressEnable = true
-			egressEnable = true
-		} else {
-			for _, v := range policy.Spec.PolicyTypes {
-				if strings.EqualFold(string(v), string(multiv1beta1.PolicyTypeIngress)) {
-					ingressEnable = true
-				} else if strings.EqualFold(string(v), string(multiv1beta1.PolicyTypeEgress)) {
-					egressEnable = true
-				}
-			}
-		}
+		var ingressEnable, egressEnable bool = getEnabledPolicyTypes(policy)
```

Comment:
how about to change that? (it is just for shorten code with same meaning, so not critical)
`ingressEnable, egressEnable := getEnabledPolicyTypes(policy)`

Reply:
Good idea! Fixed